types/react-dev-utils/formatWebpackMessages.d.ts

```diff
@@ -1,16 +1,14 @@
-declare namespace formatWebpackMessages {
-    interface WebpackMessages {
-        errors: string[];
-        warnings: string[];
-    }
-}
+import { Stats } from 'webpack';
 
 /**
  * Extracts and prettifies warning and error messages from webpack
  * [stats](https://github.com/webpack/docs/wiki/node.js-api#stats) object.
  */
 declare function formatWebpackMessages(
```

Comment:
Since parameters now are short enough might be a bit cleaner to inline it

```ts
declare function formatWebpackMessages(json: Stats.ToJsonOutput): {
```